todo-app/src/stores/TodoStore.js

```diff
@@ -0,0 +1,54 @@
+import { action, observable } from 'mobx';
+import moment from 'moment';
+
+// Data
+import TodoData from '../data/todo.json';
+
+export class TodoStore {
+  @observable todos = TodoData
+  @observable newDescription = ''
+  @observable newDeadline = moment().format('YYYY-MM-DD')
+
+  // Change the state of DONE foreach todo
+  @action
+  handelCheck = (todo) => {
+    todo.done = !todo.done
+  }
+
+  @action
+  handelDeleteItem = (todo) => {
+    this.todos = this.todos.filter((item) => {
+      return item.id !== todo.id
+    })
+  }
+
+  @action
+  handleDescriptionChange = (event) => {
```

Comment:
It looks like your `handleDescriptionChange` and `handleDateChange` functions can be abstracted into a single function. Remember DRY - don't repeat yourself.